librz/util/str_search.c

```diff
@@ -0,0 +1,280 @@
+#include <rz_util/rz_str_search.h>
+#include <rz_util/rz_utf8.h>
+#include <rz_util/rz_utf16.h>
+#include <rz_util/rz_utf32.h>
+
+static RzUtilStrScanOptions g_opt = {
+	.buf_size = 2048,
+	.max_uni_blocks = 4,
+	.min_str_length = 4
+};
+
+RZ_API void rz_detected_string_free(RzDetectedString *str) {
+	if (str) {
+		free(str->string);
+		free(str);
+	}
+}
+
+RZ_API int rz_scan_strings(RzList *list, RzBuffer *buf_to_scan,
```

Comment:
You added the structure for options, but you don't use it as a function parameter. Please do not hardcode it (at least in RzUtil) and pass it as a parameter.

Reply:
done! Now it is the first parameter